# Description: imagen create png files
#  imagen generates png files instead of gif files on a Debian system.
# Author: Arjan Oosting <arjanoosting@home.nl>
diff -urNad frown~/Manual/Makefile frown/Manual/Makefile
--- frown~/Manual/Makefile	2006-01-13 00:09:44.000000000 +0100
+++ frown/Manual/Makefile	2006-09-28 02:05:29.000000000 +0200
@@ -117,7 +117,7 @@
	hacha $(file).html
	imagen $(file)
	mkdir -p html
-	mv index.html $(file).html $(file).css $(file)[0-9]*.html $(file)[0-9]*.gif *motif.gif html
+	mv index.html $(file).html $(file).css $(file)[0-9]*.html $(file)[0-9]*.png *motif.gif html

 spell: $(file).tex
	egrep -v '$\%' $(file).tex | $(spell) $(file).spell | sort | uniq
